packages/connectors/connector-oauth2/src/utils.ts

```diff
@@ -36,6 +36,19 @@ const accessTokenResponseHandler = async (
   return result.data;
 };
 
+// eslint-disable-next-line @typescript-eslint/ban-types
+const getValue = (object: Record<string, unknown> | object, path: string) => {
+  return path.split('.').reduce<unknown>((accumulator, part: string) => {
+    // Check if the accumulator is an object and has the property
+    if (accumulator && typeof accumulator === 'object' && part in accumulator) {
+      // eslint-disable-next-line no-restricted-syntax
+      return (accumulator as Record<string, unknown>)[part];
+    }
+
+    return null;
+  }, object);
+};
```

Comment:
Maybe we can use something like [_.get](https://underscorejs.org/#get)?
cc @charIeszhao

Reply:
Oh, we have a `get` util method in the `silverhand-io/essentials` package, we can use it directly.
https://github.com/silverhand-io/essentials/blob/master/src/utilities/get.ts

Reply:
Yes, you can directly use the existing method.

Reply:
Understood!